### PR TITLE
 Adding Home page

### PR DESCRIPTION
**Pull request by thuyvy3110**
Use next-images to import image

### CODE REVIEW

File: views/theme.js [hunk 1]
```diff
@@ -96,6 +96,10 @@ const theme = {
     XXL: '22px',
     XXXL: '24px'
   },
+  image:{
+    background: "/bg.jpg",
```
Comment (DeKal) [inline views/theme.js]: add images folder

File: next.config.js [hunk 2]
```diff
@@ -1,4 +1,6 @@
 const withSass = require('@zeit/next-sass')
+const withImages = require('next-images')
```
Comment (DeKal) [inline next.config.js]: try remove this

File: next.config.js [hunk 3]
```diff
@@ -7,3 +9,5 @@ module.exports = withSass({
     localIdentName: '[local]'
   }
 })
+
+module.exports = withImages(withSass({}))
```
Comment (DeKal) [inline next.config.js]: wrong

File: package.json [hunk 4]
```diff
@@ -16,6 +16,7 @@
   "dependencies": {
     "@zeit/next-sass": "1.0.1",
     "next": "9.1.4",
+    "next-images": "^1.2.0",
```
Comment (DeKal) [inline package.json]: yarn add next-images

File: views/components/Home.js [hunk 5]
```diff
@@ -1,17 +1,90 @@
...
+
+const Overlay = styled.div`
+  padding: 6em;
+  position: relative;
+  width: 100%;
+  height: 100%;
+  min-height: 100%;
+  background-color: transparent;
+  background-color: rgba(#2c3e50, 0.3);
+  filter: progid:DXImageTransform.Microsoft.gradient(startColorstr=#4C000000,endColorstr=#4C000000);
```
Comment (DeKal) [inline views/components/Home.js]: try remove

Comment (thuyvy3110) [reply views/components/Home.js]: Removed 
min-height: 100%;
background-color: transparent;
filter: progid:DXImageTransform.Microsoft.gradient(startColorstr=#4C000000,endColorstr=#4C000000);

Replaced 
background-color: rgba(#2c3e50, 0.3); to rgba(44, 62, 80, 0.3);

File: package.json (hunk 4 above)
Comment (thuyvy3110) [reply package.json]: Removed next-images 
Added again by Yarn

File: next.config.js (hunk 3 above)
Comment (thuyvy3110) [reply next.config.js]: Removed

File: next.config.js (hunk 2 above)
Comment (thuyvy3110) [reply next.config.js]: Removed

File: views/theme.js (hunk 1 above)
Comment (thuyvy3110) [reply views/theme.js]: Added

File: next.config.js [hunk 6]
```diff
@@ -1,4 +1,5 @@
 const withSass = require('@zeit/next-sass')
+
```
Comment (DeKal) [inline next.config.js]: remove

File: views/components/Home.js [hunk 7]
```diff
@@ -1,17 +1,91 @@
...
+  }
+
+  @media screen and (max-width: 800px) {
+    font-size: 50px;
+  }
+`
+
+const TitleJob = styled.h2`
+  color: ${props => props.theme.colors.colorLight};
+  font-family: Trebuchet MS, sans-serif;
```
Comment (DeKal) [inline views/components/Home.js]: adding font to theme.js for future use. For ex `props.fonts.fontTrebuchet`

File: views/components/Home.js [hunk 8]
```diff
@@ -1,17 +1,91 @@
...
+  height: 100%;
+  background-color: rgba(44, 62, 80, 0.3);
+  zoom: 1;
+  background-image: url(${props => props.theme.image.overlay});
+  background-repeat: repeat;
+  background-attachment: fixed;
+`
+
+const Header = styled.div`
+  padding: 200px 10px 400px;
```
Comment (DeKal) [inline views/components/Home.js]: extract as a special parameter in theme `headerPadding`, it's will be easy for us to look at the theme and change how big the header is

File: views/theme.js [hunk 9]
```diff
@@ -96,6 +96,11 @@ const theme = {
     XXL: '22px',
     XXXL: '24px'
   },
+  image: {
```
Comment (DeKal) [inline views/theme.js]: images

File: next.config.js [hunk 10]
```diff
@@ -1,5 +1,4 @@
 const withSass = require('@zeit/next-sass')
-
```
Comment (DeKal) [inline next.config.js]: revert this file

File: views/components/__test__/Home.spec.tsx [hunk 11]
```diff
@@ -0,0 +1,12 @@
+import * as React from 'react'
+import { shallowWithTheme } from '~/utils/withThemeProviders'
+import Home from '~/views/components/Home'
+
+describe('Home', () => {
+
+  it('should render without throwing an error', function () {
+    const wrap = shallowWithTheme(<Home />)
```
Comment (DeKal) [inline views/components/__test__/Home.spec.tsx]: try use mountWithTheme

File: views/components/__test__/Home.spec.tsx [hunk 12]
```diff
@@ -0,0 +1,12 @@
+import * as React from 'react'
+import { shallowWithTheme } from '~/utils/withThemeProviders'
```
Comment (DeKal) [inline views/components/__test__/Home.spec.tsx]: try changing to mountWithTheme

**Pull request merged**